affiliated/affiliated_package_review_process.md

```diff
@@ -82,41 +82,53 @@ posted on the pull request and also in a reply to your email.
 
 ## Reviewing affiliated packages
 
-*These instructions are for coordinators and reviewers*
-
-One of the coordination committee members will be in charge of the review
-process (not necessarily the same person for each package). This coordination
-committee member (hereafter the *coordinator*) will find someone to carry out
-the main review of the package (hereafter the *reviewer*). In this sense the
-role  of the coordinator is similar to that of a journal editor.  However
-this is only partially true, because the coordinator (speaking for the
-coordination committee as a whole) may add their views to the review, unlike a
-journal editor who is often strictly impartial on the review itself.
-The pool of available reviewers will be anyone who has an official role on
-[the Astropy team](http://www.astropy.org/team.html).
-
-The coordinator sends out an email to possible reviewers one at a time to find
-someone who is willing to review the package, using the template at the bottom
-of this document. Let them know at this point that the coordinator will also be
-doing a review and synthesizing the results into a single review. Once a
-reviewer accepts, the coordinator sends them an email with detailed review
-instructions, using the template at the bottom of this document. The coordinator
-then leaves a message on the pull request saying:
+*These instructions are for editors and reviewers.*
+
+The review process will be overseen and coordinated by individuals that are in
+the *Affiliated package review editor* role. This role
+may be filled by one or more individuals. For the rest of this section, we
+use the singular term *editor*, although multiple people may share this role
+(and choose how to do this). The
+editor will start off by finding someone to carry out the main review of the
+package (hereafter the *reviewer*). In this sense the role of the editor is
+similar to that of a journal editor (and hence the role name). However this is
+only partially true, because here the editor may add their views to the review,
```

Comment:
```suggestion
similar to that of a journal editor (and hence the role name). However this analogy
is only partially accurate, because here the editor may add their views to the review,
```